### PR TITLE
Pipeline setup

### PR DESCRIPTION
**Pull request by tjlam**
Changes 
- added services to save video, extract librosa, and add rows to db
- added helpers to make read and write to the db, and librosa
- You may need to add some folders to the home dir: datastore/audio, datastore/video, datastore/librosas

to run it `python run.py [yt url]`

### CODE REVIEW

File: services/mfccExtractor.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import helpers.librosaHelper as librosaHelper
+
+def getMfcc(librosaPath, intervals):
+  output = []
+
+  # retrieve librosa file
+  audio, sampleRate = librosaHelper.loadLibrosa(librosaPath)
+  
+  # iterate intervals and calculate mfccs 
+  for idx, (start, end) in enumerate(intervals):
```
Comment (mattlianje) [inline services/mfccExtractor.py]: `for (start, end) in intervals:` should do it no?

File: services/createIntervals.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# helper function to format vidUrl and interval
+def formatValues(vidUrl, interval):
+  return "('{:}', {:}, {:})".format(vidUrl, interval[0], interval[1])
+
+
+# creates rows in db with interval start, end, and yt url
+def insertIntervals(vidUrl, intervals):
+  if not dbHelper.urlExists(vidUrl):
+    rowValues = list(map(lambda x: formatValues(vidUrl, x), intervals))
```
Comment (mattlianje) [inline services/createIntervals.py:29]: pythonic!

File: services/ampExtractor.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import helpers.librosaHelper as librosaHelper
+
+def getAmp(librosaPath, intervals):
+  output = []
+
+  # retrieve librosa file
+  audio, sampleRate = librosaHelper.loadLibrosa(librosaPath)
+  
+  # iterate intervals and calculate mfccs 
+  for idx, (start, end) in enumerate(intervals):
```
Comment (mattlianje) [inline services/ampExtractor.py]: same - `for (start, end) in intervals:`?

**Review by mattlianje — CHANGES_REQUESTED** [review]
Looks very elegant and concise - we could add some ASCII art or smth in run.py to visualize our depency graph of sorts but it seems that doing it with modules as you did is the right way.

Stylish - OSSU!) Just 1 nit ...

File: services/mfccExtractor.py (hunk 1 above)
Comment (tjlam) [reply services/mfccExtractor.py]: oh true

**Review by mattlianje — APPROVED** [review]
LGTM 🥋

**Pull request merged**